.github/workflows/translations-sync.yml

```diff
@@ -0,0 +1,43 @@
+name: Sync translations
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [master]
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  sync-translations:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20
+
+      - name: Generate JSON source files from Docusaurus config
+        run: npm ci && npm run write-translations
+
+      - name: Sync sources and translations
+        uses: crowdin/github-action@v2
+        with:
+          config: "crowdin.yml"
+          upload_sources: true
+          upload_translations: true
+          download_translations: true
+          export_only_approved: true
+          localization_branch_name: l10n_crowdin_translations
+          create_pull_request: true
+          pull_request_title: "New Crowdin translations"
+          pull_request_body: "New translations from Crowdin."
+          pull_request_base_branch_name: "master"
```

Comment:
@h0ngcha0 This is required so that Crowdin can open PRs in this repo. We need to make sure that GitHub Actions are allowed to do that. This can be toggled in the settings of this repo: https://github.com/alephium/docs/settings/actions

It looks like this:

![image](https://github.com/alephium/docs/assets/1579899/9d521222-7f8d-4096-b031-2963041c26c9)

Reply:
Okey, seems that I don't have permission to set it. Maybe Cheng can help with this.

Reply:
It's already the case as I just checked.